railties/lib/rails/commands/notes/notes_command.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require "rails/source_annotation_extractor"
+
+module Rails
+  module Command
+    class NotesCommand < Base # :nodoc:
+      class_option :annotations, aliases: "-a", desc: "Filter by specific annotations, e.g. Foobar TODO", type: :array, default: %w(OPTIMIZE FIXME TODO)
+
+      def perform(*)
+        require_application_and_environment!
+
+        display_deprecation_warning
+        display_annotations
+      end
+
+      private
+        def display_annotations
+          annotations = options[:annotations]
+          tag = (annotations.length > 1)
+
+          say Rails::SourceAnnotationExtractor.enumerate annotations.join("|"), tag: tag, dirs: directories
+        end
+
+        def directories
+          Rails::SourceAnnotationExtractor::Annotation.directories + (source_annotation_directories || "").split(",")
+        end
+
+        def display_deprecation_warning
+          return unless source_annotation_directories
+          say ActiveSupport::Deprecation.warn("`SOURCE_ANNOTATION_DIRECTORIES` will be deprecated in Rails 6.1. You can add default directories by using config.annotations.register_directories instead.")
+          say "\n"
```

Comment:
Can't remember, do we even need to pass a newline in here? Or would `say` just do it for us?

Reply:
You're right. `say` adds the extra line without need to be passed `\n`. I'll modify